### DISCUSSION TITLE
The installation method is in asc2022

### DISCUSSION THREAD

**Discussion by AbtMyML** [deepmodeling/deepmd-kit#1474] [Q&A]
![屏幕截图 2022-02-13 162919](https://user-images.githubusercontent.com/71914594/153745371-9a1ff0af-796e-4e08-991b-6c2849f0baef.png)
Can deepmd kit only be installed through the above command？Can't I use the conda command to install?
@njzjz

**Comment by njzjz** [comment]
Conda packages were built for [releases](https://github.com/deepmodeling/deepmd-kit/releases/). If you need what haven't been built, you need to build them by yourself.

**Reply by AbtMyML** [reply]
Can I use the conda command to install for asc2022？
@njzjz

**Reply by njzjz** [reply]
You have to build conda packages firstly. See [Building conda packages](https://docs.deepmodeling.org/projects/deepmd/en/latest/install/build-conda.html).
